.github/ISSUE_TEMPLATE/7_cherry_pick.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Request a cherry-pick.
+about: As a contributor, you would like to request that a feature be cherry-picked into a release.
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+commit(s) to merge: (Enter the shortened commit hash of the commit that has been merged to master/main. <fabfeed>)
+
+merge instructions: clean merge
```

Comment:
I don't think we can usually answer this question because we don't know what what parent it will be cherry-picked onto.

Reply:
This space is a little jumbled.  The current imagined flow is:

1. User wants to submit a cherry-pick request.
2. User fills out form.
3. Release manager notes the candidate branch in the issue comments.
4. User checks for clean merge.
     -  `git checkout <candidate-branch>` 
     -  `git cherry-pick <commit-hash>`
5. User notes if there are merge conflicts or not

Maybe there is a world where we can write an automated comment to the issue that gives merge instructions and notes the correct candidate branch.

Reply:
It can't be a required field then or you would never be able to get past 2.  I don't think it really matters if it's a clean merge or not.  If it's not a clean merge then it means the author or reviewer of the PR may need to help the cherry-picker fix up the merge conflicts, but I don't think that should be a deciding factor to whether a cherry-pick request should be accepted.  With or without merge conflicts you can still bork things badly, it still needs to be tested.

Reply:
Agree on removing this field.

I still think it is valuable to have the author of the PR test that it merges cleanly.  

I see a near future state where there will be a rotation of people releasing the Flutter SDK.  For a process that already takes days to complete, not having to wait for a response and investigation from the PR authors during the release window makes the release easier and more predictable.